README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://api.shippable.com/projects/5551c20fedd7f2c052e9809d/badge?branchName=indigo-devel)](https://app.shippable.com/projects/5551c20fedd7f2c052e9809d) [![Circle CI](https://circleci.com/gh/shadow-robot/sr_vision.svg?style=shield)](https://circleci.com/gh/shadow-robot/sr_vision) [![Build Status](https://semaphoreci.com/api/v1/projects/9cf4eebf-fb5d-465f-b3cb-268901a9d312/516822/shields_badge.svg)](https://semaphoreci.com/andriy/sr_vision) [![Documentation Status](https://readthedocs.org/projects/sr-vision/badge/?version=latest)](http://sr-vision.readthedocs.org/) [![Code Health](https://landscape.io/github/shadow-robot/sr_vision/indigo-devel/landscape.svg?style=flat)](https://landscape.io/github/shadow-robot/sr_vision/indigo-devel) [![codecov.io](http://codecov.io/github/shadow-robot/sr_vision/coverage.svg?branch=indigo-devel)](http://codecov.io/github/shadow-robot/sr_vision?branch=indigo-devel)
+[![Build Status](https://api.shippable.com/projects/5551c20fedd7f2c052e9809d/badge?branchName=kinetic-devel)](https://app.shippable.com/projects/5551c20fedd7f2c052e9809d) [![Circle CI](https://circleci.com/gh/shadow-robot/sr_vision.svg?style=shield)](https://circleci.com/gh/shadow-robot/sr_vision) [![Build Status](https://semaphoreci.com/api/v1/projects/9cf4eebf-fb5d-465f-b3cb-268901a9d312/516822/shields_badge.svg)](https://semaphoreci.com/andriy/sr_vision) [![Documentation Status](https://readthedocs.org/projects/sr-vision/badge/?version=latest)](http://sr-vision.readthedocs.org/) [![Code Health](https://landscape.io/github/shadow-robot/sr_vision/kinetic-devel/landscape.svg?style=flat)](https://landscape.io/github/shadow-robot/sr_vision/kinetic-devel)
```

Comment:
Semaphore CI badge is not working.
Also Landscape one is not working.
Please remove them.

Reply:
Fixed in the latest commit